api_app/admin.py

```diff
@@ -34,3 +34,12 @@ class JobAdminView(admin.ModelAdmin):
 class TagAdminView(admin.ModelAdmin):
     list_display = ("id", "label", "color")
     search_fields = ("label", "color")
+
+
+@admin.register(PluginCredential)
+class PluginCredentialAdminView(admin.ModelAdmin):
+    list_display = ("id", "type", "attribute", "plugin_name")
+    search_fields = (
+        "attribute",
+        "plugin_name",
+    )
```

Comment:
Django also allows you to create filters over the fields. I think filters to select the "type" and the plugin name are useful.